cumulusci/tasks/github.py

```diff
@@ -207,3 +210,51 @@ def _run_task(self):
                 )
 
                 self.logger.info('Merge conflict on branch {}: created pull request #{}'.format(branch.name, pull.number))
+
+
+class CommitApexDocs(BaseGithubTask):
+
+    task_options = {
```

Comment:
Can any of these task options receive default values from the ProjectConfig in cumulusci.yml? 

We have a config section for ApexDocs as well as one where we list git branches. It'd be nice if this task worked "automatically" if you have the github service configured and your apexdocs setup. 

I bet we could make very reasonable defaults for `branch` and `dir_local` that inherit from the global yaml